apps/designer/app/designer/features/inspector/inspector.tsx

```diff
@@ -29,6 +30,7 @@ export const Inspector = ({ publish }: InspectorProps) => {
   if (selectedInstanceData === undefined) {
     return (
       <Box css={{ p: "$2" }}>
+        {/* @todo: use this space for something more usefull: a-la figma's no instance selected sate */}
```

Comment:
maybe create an issue with a more specific proposal? Unless its completely out of beta